bases/renku_data_services/data_api/app.py

```diff
@@ -13,7 +14,7 @@
     UserResourcePoolsBP,
 )
 from renku_data_services.project.blueprints import ProjectsBP
-from renku_data_services.session.blueprints import SessionsBP
+from renku_data_services.session.blueprints import SessionEnvironmentsBP, SessionLaunchersBP
```

Comment:
I suggest `EnvironmentDP` instead of `SessionEnvironmentsBP`.

Reply:
Done in [d5fec8d](https://github.com/SwissDataScienceCenter/renku-data-services/pull/122/commits/d5fec8d6e4fee8171398ffa4d10329cdcb79bd43).